Documentation/packaging-installation.md

```diff
@@ -0,0 +1,82 @@
+# .NET Core Packaging and Installation
+
+This document helps you install or package a .NET Core SDK built using source-build.
+
+The SDK built by source-build is generally not portable. That means it will work on the same operating system where it was built. It will not work on older operating systems. It may work on newer operating systems.
+
+The built SDK is generally located at `artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz`.
+
+## Using the SDK directly (install per-user)
+
+To use the SDK directly, unpack the SDK to any directory and then use the `dotnet` executable from it.
+
+You can find more details on how to manually install an SDK from a tarball at [manually installing an SDK](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#manual-install). The same steps listed should work for all Linux distributions where `bash` is the default shell.
+
+## Installing the SDK globally
+
+If you want to install the SDK globally (and not per user), here are some suggestions.
+
+1. Extract the tarball to a distribution-appropriate location such as `/usr/local/lib64/dotnet/`, `/usr/local/lib/dotnet/`, or `/usr/local/lib/x86_64-linux-gnu/dotnet/`.
+
+2. Create a symlink from `/usr/local/bin/dotnet` (or equivalent) to where you installed the SDK in the previous step. This lets users simply run `dotnet` and have it work.
```

Comment:
Should this mention "to the `dotnet` executable in the location you installed the SDK"? I think I've seen some people tripped up by the path ending up with `.../dotnet/dotnet`. Maybe just give `/usr/local/lib64/dotnet/dotnet` as an example?

Reply:
Good idea! Thanks.